documentation/feature/medical-system.md

```diff
@@ -6,238 +6,370 @@ group: feature
 order: 4
 parent: wiki
 ---
+<p align="center">
+<b>Disclaimer</b>
+
+The documentation for the medical system being extremely long it's highly advised to use the table of contents at the top right corner of the page.
+</p>
 
 ## 1. Overview
-ACE provide users with a more realistic medical system and comes in both a basic and advanced version. This page will detail the differences between both systems and what they do. It is split into two parts; basic and advanced. Both versions have overlap but each have their own unique characteristics. 
+ACE3 provides users with a more realistic medical system and comes in both a basic and an advanced versions. This page will detail the differences between both systems and what they do as well as how to use them efficiently.
 
+### 1.1 Basic medical
+ACE3's basic medical system is quite a bit more complex than Arma 3's default system, but not really difficult to grasp. ACE3 basic medical is a mixture between the ACE2 and AGM medical systems.
+All interactions in the medical system are done with the interaction menu. Non-medics can - by default - not perform all actions (Epinephrine and IVs) and their actions take more time as when performed by trained medics.
```

Comment:
Correct either way, but like @jonpas I too would prefer the WP convention.

While we are on this sentence, splitting the `can not` verb with a sentence break fucks up the emphasis. @alganthe, may I suggest you use a joined spelling and move it after the break:
> Non-medics – by default – **cannot** perform all actions...